README.md

```diff
@@ -1,22 +1,94 @@
-# Observable Swarm
+# Production-grade Docker Swarm Cluster with Logging and Monitoring
 
-Swarm is a container scheduler that allows the end user to treat a cluster of
-units as an elastic runtime to host application containers.
+## Overview
 
-Beats are the core of data shipping in the new Elastic Stack by Elastic Co.
-Famous for their ELK stack, these succeed any agents you might be familiar with.
+This is a Docker Swarm bundle that also includes logging and monitoring. It is comprised of the following components and features:
 
-When you mash these two together, you get a nice observable solution for your
-containers.
+- Swarm (clustering engine)
+  - Two node starter cluster, tested up to 20 nodes  
```

Comment:
ships with 3 i believe. That was an arbitrary number by me, we can scale it back down to two...